web/styles/settings.css

```diff
@@ -1490,6 +1490,41 @@ $option_title_width: 180px;
     }
 }
 
+.dropdown-list-widget .dropdown-list-body,
```

Comment:
I'm unsure if reusing the styles targeted for `li a` like this is the optimal approach. I would greatly appreciate any feedback on this.